specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/stable/2022-01-01/managedcluster.json

```diff
@@ -0,0 +1,1850 @@
+            "description": "The cluster will not be automatically upgraded to the latest Service Fabric runtime version. The cluster is upgraded by setting the **clusterCodeVersion** property in the cluster resource."
+          }
+        ]
+      }
+    },
+    "IPTag": {
+      "type": "object",
+      "required": [
+        "ipTagType",
+        "tag"
+      ],
+      "properties": {
+        "ipTagType": {
+          "type": "string",
+          "description": "The IP tag type."
+        },
+        "tag": {
+          "type": "string",
+          "description": "The value of the IP tag."
+        }
+      },
+      "description": "IPTag associated with the object."
+    },
+    "LoadBalancingRule": {
+      "type": "object",
+      "required": [
+        "frontendPort",
+        "backendPort",
+        "protocol",
+        "probeProtocol"
+      ],
+      "properties": {
+        "frontendPort": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The port for the external endpoint. Port numbers for each rule must be unique within the Load Balancer. Acceptable values are between 1 and 65534.",
+          "minimum": 1,
+          "maximum": 65534
+        },
+        "backendPort": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The port used for internal connections on the endpoint. Acceptable values are between 1 and 65535.",
+          "minimum": 1,
+          "maximum": 65534
+        },
+        "protocol": {
+          "type": "string",
+          "description": "The reference to the transport protocol used by the load balancing rule.",
+          "enum": [
+            "tcp",
+            "udp"
+          ],
+          "x-ms-enum": {
+            "name": "protocol",
+            "modelAsString": true
+          }
+        },
+        "probePort": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The prob port used by the load balancing rule. Acceptable values are between 1 and 65535.",
+          "minimum": 1,
+          "maximum": 65534
+        },
+        "probeProtocol": {
+          "type": "string",
+          "description": "the reference to the load balancer probe used by the load balancing rule.",
+          "enum": [
+            "tcp",
+            "http",
+            "https"
+          ],
+          "x-ms-enum": {
+            "name": "probeProtocol",
+            "modelAsString": true
+          }
+        },
+        "probeRequestPath": {
+          "type": "string",
+          "description": "The probe request path. Only supported for HTTP/HTTPS probes."
+        },
+        "loadDistribution": {
+          "type": "string",
+          "description": "The load distribution policy for this rule."
+        }
+      },
+      "description": "Describes a load balancing rule."
+    },
+    "LongRunningOperationResult": {
+      "type": "object",
+      "properties": {
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Also add id property here. 
Refer Azure-AsyncOperation Resource format here: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#azure-asyncoperation-resource-format](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#azure-asyncoperation-resource-format)

Reply:
@RamyasreeChakka the Id parameter says "Optional. (but recommended)" we do not have it on the code. Do we need to add it?

Reply:
Not required. Didn't notice optional. Thanks for pointing it out.